### PR TITLE
[improve/Android] Should show the grey color texts while ID is under reviewing

### PR DESCRIPTION
**Pull request by Thisak**
Should show grey color text while id is under reviewing

### CODE REVIEW

**Comment by Thisak** [comment]
https://app.asana.com/0/515943528653251/728968912835310

File: app/src/main/kotlin/com/ttvnp/ttj_asset_android_client/presentation/ui/fragment/SettingsProfileEditFragment.kt [hunk 1]
```diff
@@ -271,6 +275,91 @@ class SettingsProfileEditFragment : BaseMainFragment(), SettingsProfileEditPrese
...
+                textDOB.setOnClickListener {
+                    clearUnderReviewMessage(view)
+                    textDOB.requestFocus()
+                    Toast.makeText(context, getString(R.string.message_under_review), Toast.LENGTH_SHORT).show()
+                }
+            }
+        }
+    }
+
+    private fun setEnableView(b: Boolean) {
```
Comment (m-toyoda) [inline app/src/main/kotlin/com/ttvnp/ttj_asset_android_client/presentation/ui/fragment/SettingsProfileEditFragment.kt]: @Thisak I don't like this attribute name :(

File: app/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -28,6 +28,7 @@
     <string name="app_start">始める</string>
 
     <!-- general -->
+    <string name="message_under_review">When ID was approved or under review, you can change only the avatar image.</string>
```
Comment (m-toyoda) [inline app/src/main/res/values-ja/strings.xml]: Get ja text please

File: app/src/main/res/values-vi/strings.xml [hunk 3]
```diff
@@ -29,6 +29,7 @@
     <string name="app_start">Bắt đầu</string>
 
     <!-- general -->
+    <string name="message_under_review">When ID was approved or under review, you can change only the avatar image.</string>
```
Comment (m-toyoda) [inline app/src/main/res/values-vi/strings.xml]: Get vi text please

File: app/src/main/res/values-ja/strings.xml (hunk 2 above)
Comment (Thisak) [reply app/src/main/res/values-ja/strings.xml]: Okay

File: app/src/main/res/values-vi/strings.xml (hunk 3 above)
Comment (Thisak) [reply app/src/main/res/values-vi/strings.xml]: Okay

File: app/src/main/kotlin/com/ttvnp/ttj_asset_android_client/presentation/ui/fragment/SettingsProfileEditFragment.kt (hunk 1 above)
Comment (Thisak) [reply app/src/main/kotlin/com/ttvnp/ttj_asset_android_client/presentation/ui/fragment/SettingsProfileEditFragment.kt]: okay

**Pull request merged**